.gitignore

```diff
@@ -11,6 +11,7 @@
 *.js.map
 !src/**/*.d.ts
 !ava.config.js
+!vite.config.js
```

Comment:
Just realized vite should be a ts file anyway, so no changes to this file are needed. In general, changes to .gitignore are very rare.